README.md

```diff
@@ -51,12 +51,33 @@
   <a href="https://tanka.dev/tutorial/overview"><strong>Let's kill some YAML together&nbsp;&nbsp;▶</strong></a>
 </p>
 
+Another Grafana Tanka community call is coming up!
+
+- :calendar: Mark the date: 2020-10-06 16:00 UTC
+- :tv: Join the meet in this doc: https://bit.ly/3czxbFz
+
 ## :rocket: Getting started
 
 To get started, [install Tanka](https://tanka.dev/install) first, and then
 [follow the tutorial](https://tanka.dev/tutorial/overview). This should get you
 on track quickly.
 
+## :busts_in_silhouette: Community
+
+The Tanka community is core to the project. Connect to us using the `#tanka`
```

Comment:
That sounds cheesy. What about "We're doing this so people can use it! We would love you to join us. You can use our Slack etc pp."